docs/openapi/components/schemas/common/HouseBillOfLadingCertificate.yml

```diff
@@ -0,0 +1,80 @@
+$linkedData:
+  term: HouseBillOfLadingCertificate
+  '@id': https://w3id.org/traceability#HouseBillOfLadingCertificate
+title: House Bill of Lading Certificate
+description: >- 
+  A bill of lading issued by a freight forwarder. Often covers a consignment of parcels from various shippers that has been grouped or consolidated by the forwarder. The forwarder may, for example, receive a single groupage bill of lading from the carrier, then issue a series of House B/Ls to the respective shipper.
+  (source: Olegario Llamazares: DICTIONARY OF INTERNATIONAL TRADE, Key definitions of 2000 trade terms and acronyms).
+type: object
+properties:
+  '@context':
+    type: array
+    const:
+      - 'https://www.w3.org/2018/credentials/v1'
+      - 'https://w3id.org/traceability/v1'
+  type:
+    type: array
+    const:
+      - VerifiableCredential
+      - HouseBillOfLadingCertificate
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    type: object
+  credentialSubject:
+    type: object
```

Comment:
I think you mean here... yes, in a future pass `type: object` can be replaced with `$ref`.... we have that issue all over the place.

Reply:
yes, I did mean here.
which is ok for now, and thats not a merging blocker.
What I don't get is why is it wrapping a Consignment not a Bill Of Lading, and why are both House and Master Bills of Lading identical, is that because that isn't done yet?

Reply:
BillOfLading document is evidence of contract of carriage; it describes a consignment. Ref. the CEFACT model. 
A HBL is the same as a MBL, except that it is issued by a Freight Forwarder (instead of a carrier). 
Shipping Instructions is the Shipper's input, describing to the carrier/forwarder what needs to go on the BL.